.gitignore

```diff
@@ -1,9 +1,17 @@
+.DS_*
 *.log
-.DS_Store
-node_modules
+logs
+**/*.backup.*
+**/*.back.*
 .cache
-.rts2_cache_cjs
-.rts2_cache_esm
-.rts2_cache_umd
-.rts2_cache_system
+
+node_modules
+bower_componets
```

Comment:
Delete bower_components. That is for a package manager and build tool chain from many years ago that we will not be using.